client/package.json

```diff
@@ -17,6 +17,7 @@
     "react-apollo": "^2.5.6",
     "react-final-form": "^6.1.0",
     "react-native": "0.59.9",
+    "react-native-elements": "^1.1.0",
```

Comment:
we do not use react native elements. Please remove this package and use react native components.